## 80% of end-end tests have passed
```
Total executed tests: 51
Failed tests: 6
Expected to fail tests: 4
Passed tests: 41
Not executed tests: 1
```
```
IDs of not executed tests: 702936 
```
```
IDs of failed tests: 727230,704613,703086,702894,704615,727229 
```
```
IDs of expected to fail tests: 703495,702948,727232,703629 
```
<h3>Not executed tests (1)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/status-mobile/job/e2e/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=feature/display-saved-addresses-in-send-flow&PR_ID=20418&APK_NAME=20418.apk&TR_CASE_IDS=702936">Rerun not executed tests</a></li></details><h3>Failed tests (6)</h3><details><summary>Click to expand</summary><br/><li><a href="https://ci.status.im/job/status-mobile/job/e2e/job/status-app-prs-rerun/parambuild/?BRANCH_NAME=feature/display-saved-addresses-in-send-flow&PR_ID=20418&APK_NAME=20418.apk&TR_CASE_IDS=727230,704613,703086,702894,704615,727229">Rerun failed tests</a></li><br/><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3572347">test_community_mark_all_messages_as_read</a>, id: 703086 </b></td></tr><tr><td><p><blockquote><div>Device 1: Tap on found: Button </div><div>Device 1: Click until `Text` by `accessibility id`: `community-description-text` will be presented </div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:796: in test_community_mark_all_messages_as_read
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 New messages counter is not shown in home > Community element
E    New messages counter is not shown in community channel element
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7aab1625c66c4b7691d73d2d103f1825?auth=65b7f9a19f38e536447625414b9bb1e9#506">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c1821417b6df4773ac126c42077ac1dc?auth=789941c1cab8ca6c8c63b97fc14c1302#415">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3572341">test_community_contact_block_unblock_offline</a>, id: 702894 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `ProfileButton` by `accessibility id`: `open-profile` </div><div>Device 1: Find `Button` by `accessibility id`: `icon, Messages, label-component, icon` </div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:719: in test_community_contact_block_unblock_offline
    profile_1.profile_messages_button.click()
../views/base_element.py:90: in click
    element = self.find_element()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 1: Button by accessibility id: `icon, Messages, label-component, icon` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7aab1625c66c4b7691d73d2d103f1825?auth=65b7f9a19f38e536447625414b9bb1e9#430">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c1821417b6df4773ac126c42077ac1dc?auth=789941c1cab8ca6c8c63b97fc14c1302#398">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3572359">test_community_edit_delete_message_when_offline</a>, id: 704615 </b></td></tr><tr><td><p><blockquote><div># STEP: Receiver is checking if initial messages were delivered</div><div>Device 1: Looking for a message by text: message to edit </div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:811: in test_community_edit_delete_message_when_offline
    self.channel_1.driver.fail("Message '%s' was not received" % message)
base_test_case.py:178: in fail
    pytest.fail('Device %s: %s' % (self.number, text))
 Device 1: Message 'message to edit' was not received
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7aab1625c66c4b7691d73d2d103f1825?auth=65b7f9a19f38e536447625414b9bb1e9#553">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c1821417b6df4773ac126c42077ac1dc?auth=789941c1cab8ca6c8c63b97fc14c1302#444">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestDeepLinksOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3572358">test_links_open_universal_links_from_chat</a>, id: 704613 </b></td></tr><tr><td><p><blockquote><div>Device 1: Find `Text` by `accessibility id`: `community-title` </div><div>Device 1: `Text` is `Closed community` </div></blockquote></p>

```
critical/test_deep_and_universal_links.py:71: in test_links_open_universal_links_from_chat
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Community 'e2e community token-gated' was not requested to join by the url https://status.app/c/G0IAAGRy3S0pLm1NeQ91pwKgFaFGoIlkxXmDxITXXi3KdZ7vpb9-ERczhqwoFwMK6sCPnKwA#zQ3shtJ8A3rxUZWcfWDL3vKYQBt3jMPTuf44dJUGfjZjsek66
E    Community 'Open community for e2e' was not requested to join by the url https://status.app/c/GzwAAGS9TbI9CiYaoZj0TRNelWAPwk8gSHIEmqgqb0ADPw5rnFvk6bp_n3BlZxVMshmxFyI=#zQ3shkrmSmwb66sw2EuUvC5iqiNqEeevTfjtks3p8ZaaCYoNj
```

<br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/14ca4f2266c84153a1a5cb0d717f9a0a?auth=db9b217b1727e3957586996d1d5d49f5#53">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletMultipleDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3572361">test_wallet_send_asset_from_drawer</a>, id: 727230 </b></td></tr><tr><td><p><blockquote><div># STEP: Getting ETH amount in the wallet of the sender before transaction</div><div>Device 1: Find `WalletTab` by `accessibility id`: `wallet-stack-tab` </div></blockquote></p>

```
critical/test_wallet.py:120: in test_wallet_send_asset_from_drawer
    sender_balance, receiver_balance, eth_amount_sender, eth_amount_receiver = self._get_balances_before_tx()
critical/test_wallet.py:41: in _get_balances_before_tx
    self.wallet_1.wallet_tab.click()
../views/base_element.py:90: in click
    element = self.find_element()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 1: WalletTab by accessibility id: `wallet-stack-tab` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3572360">test_wallet_send_eth</a>, id: 727229 </b></td></tr><tr><td><p><blockquote><div>Device 1: Tap on found: LogInButton </div><div>Device 1: Find `Button` by `accessibility id`: `done` </div></blockquote></p>

```
critical/test_wallet.py:111: in test_wallet_send_eth
    self.wallet_1.send_asset(address=self.receiver['address'], asset_name='Ether', amount=amount_to_send)
../views/wallet_view.py:100: in send_asset
    self.confirm_transaction()
../views/wallet_view.py:88: in confirm_transaction
    self.done_button.click()
../views/base_element.py:90: in click
    element = self.find_element()
../views/base_element.py:79: in find_element
    raise NoSuchElementException(
 Device 1: Button by accessibility id: `done` is not found on the screen; For documentation on this error, please visit: https://www.selenium.dev/documentation/webdriver/troubleshooting/errors#no-such-element-exception
```

<br/><br/></td></tr></tbody></table></details><h3>Expected to fail tests (4)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestGroupChatMultipleDeviceMergedNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3572354">test_group_chat_mute_chat</a>, id: 703495 </b></td></tr><tr><td><p><blockquote><div># STEP: Change device time so chat will be unmuted by timer</div><div>Device 2: Long press on `ChatElement` </div></blockquote></p>

```
critical/chats/test_group_chat.py:497: in test_group_chat_mute_chat
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Chat is still muted after timeout 
```

<b>[[Chat is not unmuted after expected time: https://github.com/status-im/status-mobile/issues/19627]]</b><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/97b7f4bf299542099ab320f8cbb4a8f6?auth=56142519e2802158ca72c45021aa8e00#477">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f02dec26e2f54cb19012f48f6044557e?auth=03dd73cd3eecbb98fd345bbb4124a70f#274">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8b6ddadbbfbd4ae3a78bb4941bbb4e7d?auth=2cc0c5da88b47a8fb231d8d883ec90e6#243">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3572363">test_wallet_add_remove_watch_only_account</a>, id: 727232 </b></td></tr><tr><td><p><blockquote><div>Device 1: `Text` is `0x8d2413447ff297d30bdc475f6d5cb00254685aae` </div><div>Device 1: Click system back button </div></blockquote></p>

```
critical/test_wallet.py:214: in test_wallet_add_remove_watch_only_account
    self.home_view.driver.fail(
base_test_case.py:178: in fail
    pytest.fail('Device %s: %s' % (self.number, text))
 Device 1: Incorrect address '0x8d2413447ff297d30bdc475f6d5cb00254685aae' is shown when swiping between accounts, expected one is '0x8d2413447ff297d30bdc475f6d5cb00254685aae' 
```

<b>[[Missing networks in account address, https://github.com/status-im/status-mobile/issues/20166]]</b><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e6c25aaff431455e806f6c78756a7223?auth=4a9cb956ed8554fc7cf2288831a27f9f#103">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMergedTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3572344">test_community_hashtag_links_to_community_channels</a>, id: 702948 </b></td></tr><tr><td><p><blockquote><div>Device 2: Find `Text` by `xpath`: `//android.view.ViewGroup[@content-desc='chat-item']//android.widget.TextView[contains(@text,'#cats')]` </div><div>Device 2: Looking for a message by text: it is just a message text </div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:1124: in test_community_hashtag_links_to_community_channels
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Message in community channel is not visible for user before join 
```

<b>[[Can't enter channel after community is fetched for the first time, https://github.com/status-im/status-mobile/issues/20395]]</b><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ee2b590fc88b42fb9787f90f6d93a562?auth=ba1571560e9358a82a232e918bf9ee02#304">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b8445c5fd6c34cfea185793f7d224d8b?auth=a3cc0a9963b71da2fb1ac41e8c1bcb4f#299">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3572357">test_community_join_when_node_owner_offline</a>, id: 703629 </b></td></tr><tr><td><p><blockquote><div>Device 2: Looking for community: 'open community' </div><div>Device 2: Click until `Text` by `accessibility id`: `community-description-text` will be presented </div></blockquote></p>

```
critical/chats/test_public_chat_browsing.py:1212: in test_community_join_when_node_owner_offline
    self.errors.verify_no_errors()
base_test_case.py:190: in verify_no_errors
    pytest.fail('\n '.join([self.errors.pop(0) for _ in range(len(self.errors))]))
 Message in community channel is not visible for user before join, it was indicated as not sent for the sender before he went offline 
```

<b>[[Can't enter channel after community is fetched for the first time, https://github.com/status-im/status-mobile/issues/20395]]</b><br/><br/><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ee2b590fc88b42fb9787f90f6d93a562?auth=ba1571560e9358a82a232e918bf9ee02#503">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b8445c5fd6c34cfea185793f7d224d8b?auth=a3cc0a9963b71da2fb1ac41e8c1bcb4f#487">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details><h3>Passed tests (41)</h3><details><summary>Click to expand</summary><br/><br/><h4>Class TestOneToOneChatMultipleSharedDevicesNewUi:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3572320">test_1_1_chat_emoji_send_reply_and_open_link</a>, id: 702782 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/94588a7ce49c4c5ca45617a3c729c4cb?auth=8bee28fe8b63bbeb65fcc7bac211acf4#150">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7db08be2fa28476892e28c2aef7241a2?auth=582c28a03d68a2c7c06494546267623a#120">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3572315">test_1_1_chat_text_message_delete_push_disappear</a>, id: 702733 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/94588a7ce49c4c5ca45617a3c729c4cb?auth=8bee28fe8b63bbeb65fcc7bac211acf4#557">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7db08be2fa28476892e28c2aef7241a2?auth=582c28a03d68a2c7c06494546267623a#397">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3572327">test_1_1_chat_push_emoji</a>, id: 702813 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/94588a7ce49c4c5ca45617a3c729c4cb?auth=8bee28fe8b63bbeb65fcc7bac211acf4#416">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7db08be2fa28476892e28c2aef7241a2?auth=582c28a03d68a2c7c06494546267623a#260">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3572317">test_1_1_chat_non_latin_messages_stack_update_profile_photo</a>, id: 702745 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/94588a7ce49c4c5ca45617a3c729c4cb?auth=8bee28fe8b63bbeb65fcc7bac211acf4#358">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7db08be2fa28476892e28c2aef7241a2?auth=582c28a03d68a2c7c06494546267623a#210">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3572338">test_1_1_chat_edit_message</a>, id: 702855 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/94588a7ce49c4c5ca45617a3c729c4cb?auth=8bee28fe8b63bbeb65fcc7bac211acf4#465">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7db08be2fa28476892e28c2aef7241a2?auth=582c28a03d68a2c7c06494546267623a#291">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3572353">test_1_1_chat_send_image_save_and_share</a>, id: 703391 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/94588a7ce49c4c5ca45617a3c729c4cb?auth=8bee28fe8b63bbeb65fcc7bac211acf4#496">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7db08be2fa28476892e28c2aef7241a2?auth=582c28a03d68a2c7c06494546267623a#337">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3572313">test_1_1_chat_pin_messages</a>, id: 702731 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/94588a7ce49c4c5ca45617a3c729c4cb?auth=8bee28fe8b63bbeb65fcc7bac211acf4#212">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7db08be2fa28476892e28c2aef7241a2?auth=582c28a03d68a2c7c06494546267623a#165">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3572312">test_1_1_chat_message_reaction</a>, id: 702730 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/94588a7ce49c4c5ca45617a3c729c4cb?auth=8bee28fe8b63bbeb65fcc7bac211acf4#96">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7db08be2fa28476892e28c2aef7241a2?auth=582c28a03d68a2c7c06494546267623a#85">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityOneDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3572348">test_restore_multiaccount_with_waku_backup_remove_switch</a>, id: 703133 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/bab603a90fbf4450bfd5fb05a8b26402?auth=afa0c31198e1622f4f0eed13a4ccd9b7#230">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3572316">test_community_copy_and_paste_message_in_chat_input</a>, id: 702742 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/bab603a90fbf4450bfd5fb05a8b26402?auth=afa0c31198e1622f4f0eed13a4ccd9b7#77">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3572356">test_community_discovery</a>, id: 703503 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/bab603a90fbf4450bfd5fb05a8b26402?auth=afa0c31198e1622f4f0eed13a4ccd9b7#355">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3572340">test_community_undo_delete_message</a>, id: 702869 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/bab603a90fbf4450bfd5fb05a8b26402?auth=afa0c31198e1622f4f0eed13a4ccd9b7#106">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3572335">test_community_navigate_to_channel_when_relaunch</a>, id: 702846 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/bab603a90fbf4450bfd5fb05a8b26402?auth=afa0c31198e1622f4f0eed13a4ccd9b7#53">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3572352">test_community_mute_community_and_channel</a>, id: 703382 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/bab603a90fbf4450bfd5fb05a8b26402?auth=afa0c31198e1622f4f0eed13a4ccd9b7#125">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePRTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3572345">test_activity_center_mentions</a>, id: 702957 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/192b2290a12244aa92da05edf1a972ac?auth=8a9aef4e72f7d1b77819719ef3237871#129">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/1a22a7bb2fa341ecb9e4f7a6a8296bca?auth=b05b72aa029a55b57e538b47f70430e7#102">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3572346">test_activity_center_admin_notification_accept_swipe</a>, id: 702958 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/192b2290a12244aa92da05edf1a972ac?auth=8a9aef4e72f7d1b77819719ef3237871#170">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/1a22a7bb2fa341ecb9e4f7a6a8296bca?auth=b05b72aa029a55b57e538b47f70430e7#118">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestWalletOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3572362">test_wallet_add_remove_regular_account</a>, id: 727231 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/e6c25aaff431455e806f6c78756a7223?auth=4a9cb956ed8554fc7cf2288831a27f9f#39">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMerged:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3572349">test_community_several_images_send_reply</a>, id: 703194 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7aab1625c66c4b7691d73d2d103f1825?auth=65b7f9a19f38e536447625414b9bb1e9#236">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c1821417b6df4773ac126c42077ac1dc?auth=789941c1cab8ca6c8c63b97fc14c1302#169">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3572339">test_community_one_image_send_reply</a>, id: 702859 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7aab1625c66c4b7691d73d2d103f1825?auth=65b7f9a19f38e536447625414b9bb1e9#261">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c1821417b6df4773ac126c42077ac1dc?auth=789941c1cab8ca6c8c63b97fc14c1302#230">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3572330">test_community_emoji_send_copy_paste_reply</a>, id: 702840 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7aab1625c66c4b7691d73d2d103f1825?auth=65b7f9a19f38e536447625414b9bb1e9#311">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c1821417b6df4773ac126c42077ac1dc?auth=789941c1cab8ca6c8c63b97fc14c1302#290">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3572329">test_community_message_delete</a>, id: 702839 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7aab1625c66c4b7691d73d2d103f1825?auth=65b7f9a19f38e536447625414b9bb1e9#201">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c1821417b6df4773ac126c42077ac1dc?auth=789941c1cab8ca6c8c63b97fc14c1302#140">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3572328">test_community_message_send_check_timestamps_sender_username</a>, id: 702838 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7aab1625c66c4b7691d73d2d103f1825?auth=65b7f9a19f38e536447625414b9bb1e9#159">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c1821417b6df4773ac126c42077ac1dc?auth=789941c1cab8ca6c8c63b97fc14c1302#111">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>6. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3572333">test_community_links_with_previews_github_youtube_twitter_gif_send_enable</a>, id: 702844 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7aab1625c66c4b7691d73d2d103f1825?auth=65b7f9a19f38e536447625414b9bb1e9#335">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c1821417b6df4773ac126c42077ac1dc?auth=789941c1cab8ca6c8c63b97fc14c1302#307">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>7. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3572332">test_community_message_edit</a>, id: 702843 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7aab1625c66c4b7691d73d2d103f1825?auth=65b7f9a19f38e536447625414b9bb1e9#177">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c1821417b6df4773ac126c42077ac1dc?auth=789941c1cab8ca6c8c63b97fc14c1302#128">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>8. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3572331">test_community_unread_messages_badge</a>, id: 702841 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7aab1625c66c4b7691d73d2d103f1825?auth=65b7f9a19f38e536447625414b9bb1e9#400">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/c1821417b6df4773ac126c42077ac1dc?auth=789941c1cab8ca6c8c63b97fc14c1302#368">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityCenterContactRequestMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3572319">test_add_contact_field_validation</a>, id: 702777 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/63b587f1046e4238aed8f49dbaf9e1cb?auth=71f01593b4ba421632e70fab8b3ee0fe#135">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ccc69e57dcbe495386219a45dd891202?auth=731fbe5d5da446bd4a5b0b50cb17891d#212">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3572337">test_activity_center_contact_request_accept_swipe_mark_all_as_read</a>, id: 702851 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/63b587f1046e4238aed8f49dbaf9e1cb?auth=71f01593b4ba421632e70fab8b3ee0fe#100">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ccc69e57dcbe495386219a45dd891202?auth=731fbe5d5da446bd4a5b0b50cb17891d#118">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3572336">test_activity_center_contact_request_decline</a>, id: 702850 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/63b587f1046e4238aed8f49dbaf9e1cb?auth=71f01593b4ba421632e70fab8b3ee0fe#57">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ccc69e57dcbe495386219a45dd891202?auth=731fbe5d5da446bd4a5b0b50cb17891d#56">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestCommunityMultipleDeviceMergedTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3572326">test_community_markdown_support</a>, id: 702809 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ee2b590fc88b42fb9787f90f6d93a562?auth=ba1571560e9358a82a232e918bf9ee02#185">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b8445c5fd6c34cfea185793f7d224d8b?auth=a3cc0a9963b71da2fb1ac41e8c1bcb4f#175">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3572323">test_community_mentions_push_notification</a>, id: 702786 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ee2b590fc88b42fb9787f90f6d93a562?auth=ba1571560e9358a82a232e918bf9ee02#136">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b8445c5fd6c34cfea185793f7d224d8b?auth=a3cc0a9963b71da2fb1ac41e8c1bcb4f#117">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3572334">test_community_leave</a>, id: 702845 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/ee2b590fc88b42fb9787f90f6d93a562?auth=ba1571560e9358a82a232e918bf9ee02#301">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/b8445c5fd6c34cfea185793f7d224d8b?auth=a3cc0a9963b71da2fb1ac41e8c1bcb4f#277">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestDeepLinksOneDevice:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3572318">test_links_deep_links</a>, id: 702775 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/14ca4f2266c84153a1a5cb0d717f9a0a?auth=db9b217b1727e3957586996d1d5d49f5#200">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestGroupChatMultipleDeviceMergedNewUI:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3572314">test_group_chat_pin_messages</a>, id: 702732 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/97b7f4bf299542099ab320f8cbb4a8f6?auth=56142519e2802158ca72c45021aa8e00#357">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f02dec26e2f54cb19012f48f6044557e?auth=03dd73cd3eecbb98fd345bbb4124a70f#231">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8b6ddadbbfbd4ae3a78bb4941bbb4e7d?auth=2cc0c5da88b47a8fb231d8d883ec90e6#229">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3572351">test_group_chat_send_image_save_and_share</a>, id: 703297 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/97b7f4bf299542099ab320f8cbb4a8f6?auth=56142519e2802158ca72c45021aa8e00#283">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f02dec26e2f54cb19012f48f6044557e?auth=03dd73cd3eecbb98fd345bbb4124a70f#169">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8b6ddadbbfbd4ae3a78bb4941bbb4e7d?auth=2cc0c5da88b47a8fb231d8d883ec90e6#142">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3572350">test_group_chat_reactions</a>, id: 703202 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/97b7f4bf299542099ab320f8cbb4a8f6?auth=56142519e2802158ca72c45021aa8e00#174">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f02dec26e2f54cb19012f48f6044557e?auth=03dd73cd3eecbb98fd345bbb4124a70f#107">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8b6ddadbbfbd4ae3a78bb4941bbb4e7d?auth=2cc0c5da88b47a8fb231d8d883ec90e6#87">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>4. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3572324">test_group_chat_join_send_text_messages_push</a>, id: 702807 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/97b7f4bf299542099ab320f8cbb4a8f6?auth=56142519e2802158ca72c45021aa8e00#157">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f02dec26e2f54cb19012f48f6044557e?auth=03dd73cd3eecbb98fd345bbb4124a70f#79">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8b6ddadbbfbd4ae3a78bb4941bbb4e7d?auth=2cc0c5da88b47a8fb231d8d883ec90e6#79">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>5. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3572325">test_group_chat_offline_pn</a>, id: 702808 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/97b7f4bf299542099ab320f8cbb4a8f6?auth=56142519e2802158ca72c45021aa8e00#323">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/f02dec26e2f54cb19012f48f6044557e?auth=03dd73cd3eecbb98fd345bbb4124a70f#204">Steps, video, logs</a></li></ul></p><p>Device 3:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8b6ddadbbfbd4ae3a78bb4941bbb4e7d?auth=2cc0c5da88b47a8fb231d8d883ec90e6#202">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestOneToOneChatMultipleSharedDevicesNewUiTwo:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3572322">test_1_1_chat_delete_via_long_press_relogin</a>, id: 702784 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7ee965d601714f6f9ed02f5ee5afb01e?auth=cc8a25757496d732bf48f85981e1cb19#175">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/876db4f0e6e5414bb601948854d09c11?auth=09adf54b6db5046967a5e0c42513893f#126">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>2. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3572321">test_1_1_chat_is_shown_message_sent_delivered_from_offline</a>, id: 702783 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7ee965d601714f6f9ed02f5ee5afb01e?auth=cc8a25757496d732bf48f85981e1cb19#96">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/876db4f0e6e5414bb601948854d09c11?auth=09adf54b6db5046967a5e0c42513893f#85">Steps, video, logs</a></li></ul></p></ul></p></td></tr><tr><td><b>3. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3572355">test_1_1_chat_mute_chat</a>, id: 703496 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/7ee965d601714f6f9ed02f5ee5afb01e?auth=cc8a25757496d732bf48f85981e1cb19#111">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/876db4f0e6e5414bb601948854d09c11?auth=09adf54b6db5046967a5e0c42513893f#113">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table><h4>Class TestActivityMultipleDevicePR:</h4><table style="width: 100%"><colgroup><col span="1" style="width: 20%;"><col span="1" style="width: 80%;"></colgroup><tbody><tr></tr><tr><td><b>1. <a href="https://ethstatus.testrail.net/index.php?/tests/view/3572343">test_activity_center_reply_read_unread_delete_filter_swipe</a>, id: 702947 </b></td></tr><tr><td><ins>Device sessions</ins><p><ul><p>Device 1:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/8a573e9640024ac7b3025383262cf0ea?auth=30a42473d0bce6b8342fcd36d370eae3#129">Steps, video, logs</a></li></ul></p><p>Device 2:<ul><li><a href="https://eu-central-1.saucelabs.com/jobs/82f99404554d4062adf0f88a4f385aac?auth=ba7604099d5903f044a5ddfd0988f546#102">Steps, video, logs</a></li></ul></p></ul></p></td></tr></tbody></table></details>

--- a/src/status_im/contexts/wallet/send/select_address/tabs/view.cljs
+++ b/src/status_im/contexts/wallet/send/select_address/tabs/view.cljs
@@ -46,16 +46,56 @@
                                                :stack-id  :screen/wallet.select-address}])}]))
             recent-recipients))))
 
+(defn- saved-address
+  [{:keys [name address chain-short-names customization-color ens? ens]}]
+  (let [full-address           (str chain-short-names address)
+        on-press-saved-address (rn/use-callback
+                                #(rf/dispatch
+                                  [:wallet/select-send-address
+                                   {:address   full-address
+                                    :recipient full-address
+                                    :stack-id  :screen/wallet.select-address}])
+                                [full-address])]
+    [quo/saved-address
+     {:user-props      {:name                name
+                        :address             full-address
+                        :ens                 (when ens? ens)
+                        :customization-color customization-color}
+      :container-style {:margin-horizontal 8}
+      :on-press        on-press-saved-address}]))
+
+(defn- saved-addresses
+  [theme]
+  (let [group-saved-addresses (rf/sub [:wallet/grouped-saved-addresses])
+        section-header        (rn/use-callback
+                               (fn [{:keys [title index]}]
+                                 [quo/divider-label
+                                  {:tight?          true
+                                   :container-style (when (pos? index) {:margin-top 8})}
+                                  title]))
+        empty-state-component (rn/use-callback
+                               (fn []
+                                 [quo/empty-state
+                                  {:title       (i18n/label :t/no-saved-addresses)
+                                   :description (i18n/label
+                                                 :t/you-like-to-type-43-characters)
+                                   :image       (resources/get-themed-image :sweating-man
+                                                                            theme)}])
+                               [theme])]
+    [rn/section-list
+     {:key-fn                          :title
+      :shows-vertical-scroll-indicator false
+      :render-section-header-fn        section-header
+      :sections                        group-saved-addresses
+      :render-fn                       saved-address
+      :empty-component                 empty-state-component}]))
+
 (defn view
   [{:keys [selected-tab]}]
   (let [theme (quo.theme/use-theme)]
     (case selected-tab
       :tab/recent      [recent-transactions theme]
-      :tab/saved       [quo/empty-state
-                        {:title           (i18n/label :t/no-saved-addresses)
-                         :description     (i18n/label :t/you-like-to-type-43-characters)
-                         :image           (resources/get-themed-image :sweating-man theme)
-                         :container-style style/empty-container-style}]
+      :tab/saved       [saved-addresses theme]
       :tab/contacts    [quo/empty-state
                         {:title           (i18n/label :t/no-contacts)
                          :description     (i18n/label :t/no-contacts-description)

--- a/src/status_im/contexts/wallet/send/select_address/tabs/view.cljs
+++ b/src/status_im/contexts/wallet/send/select_address/tabs/view.cljs
@@ -73,7 +73,7 @@
                                   {:tight?          true
                                    :container-style (when (pos? index) {:margin-top 8})}
                                   title]))
-        empty-state-component (rn/use-callback
+        empty-state-component (rn/use-memo
                                (fn []
                                  [quo/empty-state
                                   {:title       (i18n/label :t/no-saved-addresses)

